docs/content/concepts/video.md

```diff
@@ -0,0 +1,76 @@
+This is also very simple to do, and the Python logging SDK has built-in support for it using [`Image.compress`](https://ref.rerun.io/docs/python/0.18.2/common/archetypes/#rerun.archetypes.Image.compress).
+
+Finally, you can encode the images as a video file, and log it using `AssetVideo`.
+This gives the best compression ratio, reducing file sizes and bandwidth requirements.
+
+snippet: archetypes/video_auto_frames
+
+## `AssetVideo` limitations
+Video support is new in Rerun, and has several limitations:
+
+* [#7298](https://github.com/rerun-io/rerun/issues/7298): Video playback only works in the web viewer
+* [#7354](https://github.com/rerun-io/rerun/issues/7354): Only the MP4 container format is supported
+* [#5181](https://github.com/rerun-io/rerun/issues/5181): There is no audio support
+* There is no video encoder in the Rerun SDK, so you need to create the video file yourself
+* Only a limited sets of codecs are supported (see below)
+
+## Web viewer support
+As of writing, playback of `AssetVideo` is only supported on the web viewer.
+Native video playback is coming, and can be tracked [in this GitHub issue](https://github.com/rerun-io/rerun/issues/7298).
+
+Video playback is done using the browser's own video decoder, so the supported codecs depend on your browser.
+
+Overall, we recommend using Chrome or another Chromium-based browser, as it seems to have the best video support as of writing.
+
+When choosing a codec, we recommend [AV1](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Video_codecs#av1), as that seems to have the best overall playback support. Since AV1 is patent-free, it is also likely the first codec we will support in the native viewer.
+
+For decoding video in the Web Viewer, we use the [WebCodecs API](https://developer.mozilla.org/en-US/docs/Web/API/WebCodecs_API).
+This API enables us to take advantage of the browser's hardware accelerated video decoding capabilities.
+It is implemented by all modern browsers, but with varying levels of support for different codecs, and varying levels of quality.
+
+With that in mind, here are the browsers which we have tested and verified to work:
+
+|            | Linux  | macOS | Windows |
+| ---------- | ------ | ----- | ------- |
+| Firefox    | ✅[^1] | ✅    | ✅      |
+| Chrome[^2] | ✅     | ✅    | ✅[^3]  |
+| Safari     |        | ✅    |         |
+
+[^1]: Firefox on Linux has been observed to [stutter when playing back H.264 video](https://github.com/rerun-io/rerun/issues/7532)
+[^2]: Any Chromium-based browser should work, but we don't test all of them.
+[^3]: Chrome on Windows has also been observed to stutter when playing back H.264 video. It can be mitigated by [using software rendering](https://rerun.io/docs/getting-started/troubleshooting#video-stuttering)
```

Comment:
```suggestion
[^3]: Chrome on Windows has also been observed to stutter on playback. It can be mitigated by [using software decoding](https://rerun.io/docs/getting-started/troubleshooting#video-stuttering)
```

stuttering on my machine was 100% universal on all videos I tried no matter the codec :(
changing `rendering` to `decoding` here because Rerun itself can do software rendering on native